src/controllers/authContoller.ts

```diff
@@ -445,3 +457,48 @@ export const updateProfile = async (
     return res.status(500).json({ message: 'Internal server error' });
   }
 };
+
+export const getBatches = async (
+  req: Request,
+  res: Response
+): Promise<unknown> => {
+  try {
+    const batches = await prisma.batch.findMany({
+      select: {
+        batch: true
+      }
+    });
+    return res.status(200).json(batches);
+  } catch (error) {
+    return res.status(500).json({ message: 'Internal server error' });
+  }
+};
+
+export const updateBatch = async (
```

Comment:
we can reuse the update profile route itself right ?
cuz ppl have to update their gender too, we have assigned `Male` for all the users